### ISSUE TITLE
bitbucket schema weirdness

### ISSUE THREAD

**Issue by blade2005** [python-jsonschema/check-jsonschema#440]
Bitbucket schemas aren't officially released but can be found in repo's that interact with the bitbucket pipeline config file.

The 2 schemas I can find are 
- [intellij-bitbucket-references-plugin](https://bitbucket.org/atlassianlabs/intellij-bitbucket-references-plugin/raw/master/src/main/resources/schemas/bitbucket-pipelines.schema.json)

- [atlascode](https://bitbucket.org/atlassianlabs/atlascode/raw/main/resources/schemas/pipelines-schema.json)

In the catalog is the one from atlascode. However it looks like intellij-bitbucket-references-plugin has a newer one that conforms to the documentation from bitbucket's [doc site](https://support.atlassian.com/bitbucket-cloud/docs/bitbucket-pipelines-configuration-reference/). I'll be honest I don't know which is 'official' or most up to date, but looking at the issues open with each project, I see a faster response on the intellij one for schema updates. However atlascode, looks like it is an official plugin. 

In summary intellij gets more updates and faster responses, but is a 3rd party repo; atlascode, is official but I see issues related to out of date schemas open from at least the beginning of the year. I'd suggest switching which url is in catalog to the one from intellij.

**Comment by sirosen** [comment]
Both appear to be maintained under Atlassian Labs, but it's not clear if that designation means that they are in any way official? This is, indeed, pretty weird. Like you say, it looks like one is Atlassian-driven and one is 3rd party.
SchemaStore is using the newer one ( https://github.com/SchemaStore/schemastore/pull/3098 ), and I have no particular reason to prefer one source over the other. This situation makes me a little bit uncomfortable.

I think I will take a (new, as of right now) policy stance that until/unless there is some significant reason to diverge, I'll follow SchemaStore's lead on where to source things. That at least sets a rule which I can follow and we'll see where it leads.

**Comment by blade2005** [comment]
I think that's a good direction of following schemastore's lead.
